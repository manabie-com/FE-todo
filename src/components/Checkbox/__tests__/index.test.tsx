import * as React from 'react';
import { render } from '@testing-library/react';

import { Checkbox } from '../index';

describe('<Input />', () => {
  it('should match snapshot', () => {
    const checkbox = render(<Checkbox id="test" />);
    expect(checkbox.container.firstChild).toMatchSnapshot();
  });
});
