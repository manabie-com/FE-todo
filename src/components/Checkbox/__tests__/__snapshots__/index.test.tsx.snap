// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Input /> should match snapshot 1`] = `
.c0 {
  position: relative;
  min-width: 24px;
  min-height: 24px;
}

.c0::before {
  content: '';
  position: absolute;
  z-index: -1;
  box-sizing: border-box;
  width: 24px;
  height: 24px;
  border-radius: 4px;
  border: 1px solid #2185d0;
}

.c3 {
  width: 24px;
  color: #2185d0;
  position: absolute;
  z-index: -1;
}

.c1 {
  position: absolute;
  opacity: 0;
  width: 24px;
  height: 24px;
  margin: 0;
}

.c1 ~ .c2 {
  display: none;
}

.c1:checked ~ .c2 {
  display: block;
}

.c4 {
  padding-left: 36px;
  line-height: 24px;
}

<div
  class="c0"
>
  <input
    class="c1"
    id="test"
    type="checkbox"
  />
  <svg
    class="c2 c3"
  >
    check.svg
  </svg>
  <label
    class="c4"
    for="test"
  />
</div>
`;
